tap_redshift/__init__.py

```diff
@@ -47,6 +47,9 @@
 DATETIME_TYPES = {'timestamp', 'timestamptz',
                   'timestamp without time zone', 'timestamp with time zone'}
 
+GEOMETRY = {'geometry'}
+SELECT_FORMAT = { 'symon.geo': 'ST_AsEWKT("{}")' }
```

Comment:
what does this mean? :/

Reply:
normally its some unreadable string, this makes it string readable "POINT(1 2)" or whatever other format like "POLYGON(...)"
https://postgis.net/docs/ST_AsEWKT.html